src/modules/ekf2/EKF/aid_sources/gnss/gps_checks.cpp

```diff
@@ -88,49 +82,32 @@ void Ekf::collect_gps(const gnssSample &gps)
 		_gpos_origin_eph = gps.hacc;
 		_gpos_origin_epv = gps.vacc;
 
-		_information_events.flags.gps_checks_passed = true;
-		ECL_INFO("GPS checks passed");
-	}
+		_earth_rate_NED = calcEarthRateNED(math::radians(static_cast<float>(gps.lat)));
 
-	if ((isTimedOut(_wmm_gps_time_last_checked, 1e6)) || (_wmm_gps_time_last_set == 0)) {
-		// a rough 2D fix is sufficient to lookup declination
-		const bool gps_rough_2d_fix = (gps.fix_type >= 2) && (gps.hacc < 1000);
+		_information_events.flags.gps_checks_passed = true;
```

Comment:
Shouldn't the be next to the `_gps_checks_passed = true` line?

Reply:
Yes it should be all together. 
https://github.com/PX4/PX4-Autopilot/blob/398f00ae35159bdec63dd37677cf3333fce0666a/src/modules/ekf2/EKF/aid_sources/gnss/gps_control.cpp#L67-L70